### ISSUE TITLE
New zwave-js config version available: 7.3.1-20210503

### ISSUE THREAD

**Issue by trentjw** [hassio-addons/addon-zwavejs2mqtt#162]
![update](https://user-images.githubusercontent.com/12220533/117522935-4d6cae00-af73-11eb-9e6b-d98332d4619f.png)
# Problem/Motivation

> update persistent does not install
## Expected behavior

> (server to update when pressing the install button)

## Actual behavior

>Nothing happens

zwavejs2mqtt: 4.0.1
zwave-js: 7.3.0
home id: 2165408480
home hex:0x811182e0

persistent
New zwave-js config version available: 7.3.1-20210503

press install this shows on the log

2021-05-07 20:29:00.731 INFO ZWAVE: Node 19: value updated: 37-0-currentValue true => true
2021-05-07 20:29:01.793 INFO ZWAVE: Success zwave api call installConfigUpdate { success: true, message: 'Success zwave api call', result: false }

## Steps to reproduce

> (click the install button)

## Proposed changes

> (If you have a proposed change, workaround or fix,
> describe the rationale behind it)

**Comment by cdemi** [comment]
Looks like this is related to #140. I have this issue as well:

```
2021-05-09 23:50:01.450 INFO ZWAVE: Calling api installConfigUpdate with args: [ [length]: 0 ]
2021-05-09T21:50:03.645Z DRIVER Config update failed: Package manager exited with code undefined
2021-05-09 23:50:03.647 INFO ZWAVE: Success zwave api call installConfigUpdate { success: true, message: 'Success zwave api call', result: false }
```

**Comment by DayRunner131** [comment]
Same issue here. Also related to #160 .

**Comment by peterbaker** [comment]
Same. Fresh install, have only ever been on 0.14.1

"New zwave-js config version available: 7.3.1-20210503"

Clicking install shows a "Successfully call api installConfigUpdate" dialog, but nothing happens and the config notification remains.

**Comment by yaggermr** [comment]
zwavejs2mqtt: 4.0.1  
zwave-js: 7.3.0
Home Assistant 2021.5.2
Using the update GUI install button doesn't update the zwave-js version and the notification remains.  After clicking install  a message is displayed  " successful call api installConfigUpdate" 

"New zwave-js config version available: 7.3.1-20210503."

**Comment by akennerly** [comment]
Config update persistence was fixed in the current release version of ZwaveJS2mqtt.

@frenck has already committed the new version to the add-on in #169. I suspect we'll see the new add-on version available soon. 

https://github.com/hassio-addons/addon-zwavejs2mqtt/pull/169

**Comment by frenck** [comment]
Resolved in v0.15.0
